src/app/mypage/components/Title/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { BackgroundDrawing, Underline } from '@/components/Common';
+
+export default function Title() {
+  return (
+    <article className="w-full text-65 font-luckiest flex flex-col justify-center items-center mt-100">
+      <div className="relative leading-tight">
+        <p className="">Your</p>
+        <BackgroundDrawing className="absolute-background-drawing w-290 h-100" />
+      </div>
+      <div className="relative flex flex-col justify-center items-center leading-tight">
+        <p>playground</p>
+        <Underline className="absolute bottom-2 w-500" />
```

Comment:
```suggestion
      <div className="relative text-center">
        <p>playground</p>
        <Underline className="absolute bottom-2 w-500" />
```

Underline이 absolute로 포지셔닝되고 있으므로 위와 같은 코드도 정상 작동할 것 같아요.

Reply:
글씨와 underline이 가운데 정렬되려면 flex-col 이 있어야 정렬되는 것 같습니다. justify-center는 불필요하여 삭제했습니다!